molecule/default/create.yml

```
@@ -38,6 +38,14 @@
         from_port: "{{ default_ssh_port }}"
         to_port: "{{ default_ssh_port }}"
         cidr_ip: "0.0.0.0/0"
+      - proto: tcp
+        from_port: 443
+        to_port: 443
+        cidr_ip: "0.0.0.0/0"
+      - proto: tcp
+        from_port: "0"
+        to_port: "65535"
```

Doesn't that work with molecule?

```suggestion
      - proto: all
```

---

it doesn't look like a supported value
https://docs.ansible.com/ansible/latest/collections/amazon/aws/ec2_group_module.html#parameter-rules/proto

---

It is, check in the examples on that same page. (I've also used it succesfully before)